numpy/core/function_base.py

```diff
@@ -186,7 +186,7 @@ def logspace(start, stop, num=50, endpoint=True, base=10.0, dtype=None,
     """
     Return numbers spaced evenly on a log scale.
 
-    In linear space, the sequence starts at ``base ** start``
+    In log space, the sequence starts at ``base ** start``
```

Comment:
Is this really a typo? The original seems correct to me since the exponents proper are the sequence bounds in log space, right?

```python
>>> np.logspace(1, 3, num=3)
array([  10.,  100., 1000.])
>>> np.log10(np.logspace(1, 3, num=3))
array([1., 2., 3.])
```

Reply:
I agree - this point is also more clearly expanded upon in the `Notes` section. If anything, it might be worth moving that info from the Notes up to the extended summary.